procedures/MT/rotator/rotator_performance.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>Again, probably worth mentioning here why this command is run.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>